controllers/platform/k8s.go

```diff
@@ -196,7 +196,7 @@ func createDeployment(ctx context.Context, client client.Client, platform *opera
 
 func createService(ctx context.Context, client client.Client, platform *operatorapi.SonataFlowPlatform, ps services.Platform) error {
 	lbl := map[string]string{
-		workflowproj.LabelApp: platform.Name,
+		workflowproj.LabelApp: ps.GetServiceName(),
```

Comment:
rather than change the existing label, we should probably create an additional label specific to the service.

Reply:
Should we maintain the same label for both services?

By looking at the [previous](https://github.com/flows-examples/techpreview2/blob/main/infra/dataindex/02-dataindex.yaml#L11) deployment of data-index, the label was `app.kubernetes.io/name: data-index-service-postgresql` 
The [label](https://github.com/flows-examples/techpreview2/blob/main/infra/dataindex_and_jobservice/03-jobservice.yml#L14) for the job-service was `app.kubernetes.io/name: jobs-service-postgresql` 

If we wish to maintain a relationship between the services and the platform, ownerReference seems a better representation of the relations between them.

Raised my concern, but will act based on your decision here.

Reply:
that may be the case for the tech preview templates, but the operator doesn't set that label. it sets `app: <platform.name>`. i'm suggesting we leave that label as-is and add a new service-specific label... maybe`sonataflow.org/service`?

so the labels would look like this for the related service objects -
`sonataflowplatform.name = test`

data index objects
```
  labels:
    app: test
    sonataflow.org/service: test-data-index-service
```
job service objects
```
  labels:
    app: test
    sonataflow.org/service: test-jobs-service
```